libsentrykube/git.py

```diff
@@ -1,21 +1,62 @@
-def go_to_main() -> None:
-    """
-    Switches to main stashes the current work.
-    TODO: Add arguments
-    """
-    raise NotImplementedError
-
-
-def pull_main() -> None:
-    """
-    Pull main
-    TODO: Add arguments
-    """
-    raise NotImplementedError
-
-
-def create_branch() -> None:
-    """
-    TODO: Add arguments
-    """
-    raise NotImplementedError
+import git
+
+
+class RepoNotCleanException(Exception):
+    pass
+
+
+class Git:
+    def __init__(self, repo_path: str):
+        self.repo = git.Repo(repo_path)
+        if "main" in self.repo.heads:
+            self.default_branch = "main"
+        else:
+            self.default_branch = "master"
+
+    def switch_to_default_branch(self, *, force: bool) -> None:
+        if self.repo.active_branch.name == self.default_branch:
+            return
+
+        # check if there are any changes
+        if self.repo.is_dirty():
+            if not force:
+                print("Repo not clean. Cannot switch to default branch")
```

Comment:
Please let's not add print on the stdout inside the libsentrykube module. This is supposed to be a reusable library. The presentaiton is managed by the snetry_kube module/cli module